cli/api.py

```diff
@@ -19,7 +19,13 @@ def _url(endpoint: str) -> str:
 
 
 def _error_message(response: requests.Response):
-    return response.json()['_error']['message']
+    try:
+        return response.json()['_error']['message']
```

Comment:
how about add that `if status > 500` here too, to prepend server (internal error) message with "something, hey, upload failed" ?